passage/src/main/java/id/passage/android/PassageOneTimePasscode.kt

```diff
@@ -0,0 +1,99 @@
+package id.passage.android
+
+import id.passage.android.api.LoginAPI
+import id.passage.android.api.OTPAPI
+import id.passage.android.api.RegisterAPI
+import id.passage.android.exceptions.OneTimePasscodeActivateException
+import id.passage.android.exceptions.OneTimePasscodeLoginException
+import id.passage.android.exceptions.OneTimePasscodeRegisterException
+import id.passage.android.model.ActivateOneTimePasscodeRequest
+import id.passage.android.model.AuthResult
+import id.passage.android.model.LoginOneTimePasscodeRequest
+import id.passage.android.model.RegisterOneTimePasscodeRequest
+import okhttp3.OkHttpClient
+
+class PassageOneTimePasscode(
+    private val passageClient: OkHttpClient,
+    private val tokenStore: PassageTokenStore,
+) {
+    /**
+     * Register with One Time Passcode
+     *
+     * Create a user and send a registration email or SMS to the user. The user will receive an
+     * email or text with a one time passcode to complete their registration.
+     * @param identifier valid email or E164 phone number
+     * @return OneTimePasscode
+     * @throws OneTimePasscodeRegisterException
+     */
+    suspend fun register(identifier: String): OneTimePasscode {
```

Comment:
Need to add language parameter:
```suggestion
    suspend fun register(
        identifier: String,
        language: String? = null
    ): OneTimePasscode {
```